tests/foreman/ui/test_ldap_authentication.py

```diff
@@ -1406,3 +1406,68 @@ def test_verify_attribute_of_users_are_updated(session, ldap_data, ldap_tear_dow
         assert ldap_data['ldap_user_name'] in user_values['user']['firstname']
         assert ldap_data['ldap_user_name'] in user_values['user']['lastname']
         assert ldap_data['ldap_user_name'] in user_values['user']['mail']
+
+
+@tier2
+def test_verify_ldap_filters_ipa(session, ipa_data, ldap_tear_down):
+    """Verifying ldap filters in authsource to restrict access
+
+    :id: 0052b272-08b1-11eb-80c6-0c7a158cbff4
+
+    :Steps:
+        1. Create authsource with onthefly enabled and ldap filter
+        2. Verify login from users according to the filter
+
+    :expectedresults: Login fails for restricted user
+
+    """
+    result = ssh.command(
+        cmd=f"echo {settings.ipa.password_ipa} | kinit admin", hostname=settings.ipa.hostname_ipa
+    )
```

Comment:
I see we have same command inside fixture in `pytest_fixtures/satellite_auth.py`.
Is it time to start refactoring by extracting commonly-used commands into helpers or smaller fixtures?

Reply:
We could do that, yes.